cmd.c

```diff
@@ -843,6 +843,75 @@ void cmd_syspage(char *s)
 }
 
 
+void cmd_phfs(char *s)
+{
+	u16 argsc = 0;
+	unsigned int i, major, minor, pos = 0;
+	char args[5][LINESZ + 1];
+
+	for (i = 0; argsc < 5; ++i) {
+		if (cmd_getnext(s, &pos, ". \t", NULL, args[i], sizeof(args[i])) == NULL || *args[i] == 0)
+			break;
+		argsc++;
+	}
+
+	if (argsc < 3 || argsc > 4) {
+		plostd_printf(ATTR_ERROR, "\nWrong arguments!!\n");
+		return;
+	}
+
+	/* Get major/minor */
+	major = plostd_ahtoi(args[1]);
+	minor = plostd_ahtoi(args[2]);
+
+	if (phfs_regDev(args[0], major, minor, (argsc == 3) ? NULL : args[3]) < 0)
+		plostd_printf(ATTR_ERROR, "\n%s is not registered!\n", args[0]);
+}
+
+
+void cmd_devs(char *s)
+{
+	unsigned int pos = 0;
+
+	cmd_skipblanks(s, &pos, DEFAULT_BLANKS);
+	s += pos;
+
+	if (*s) {
+		plostd_printf(ATTR_ERROR, "\nCommand should not consist of any args\n");
+		return;
+	}
+
```

Comment:
"Command foobar does not take any arguments" perhaps?

We might ignore any additional arguments, shouldn't do us any harm.

Reply:
I assume that passing wrong arguments is an incorrect usage of command.

Reply:
Ok, I have no problem with that ;)